### ISSUE TITLE
Is there a way to use this with the homekit integration?

### ISSUE THREAD

**Issue by brandoncc** [tomwpublic/hubitat_aladdinConnect#15]
First, thanks for this plugin!

I'd like to add the garage doors to the homekit integration, is there a known way to do this?

**Comment by tomwpublic** [comment]
I recommend posting on the Hubitat forums to see if anyone there has any suggestions.

**Comment by brandoncc** [comment]
Thanks for the recommendation. [We figured out what is missing](https://community.hubitat.com/t/should-the-garagedooropener-capability-integrate-with-homekit/125555/18), which is the "ContactSensor" capability.

The generic built-in garage door opener has the following capabilities and attributes:

![image](https://github.com/tomwpublic/hubitat_aladdinConnect/assets/543973/a929b68f-0533-411d-8b74-a82df787e488)

I added the `ContactSensor` capability in the driver editor, which did make the homekit integration work. Unfortunately, the driver is constantly out of sync with the actual door, so I can't use it. I thought the websocket connection would make the hub know what the door status was quickly, but even 30 seconds later the status would still be wrong (opening/closing status). Another issue that I have is that it seems to think the door is open when it is closed and closed when it is open 🤷
